data/transition-sites/dfe_graduatetalentpoolsearch.yml

```
@@ -0,0 +1,7 @@
+---
+site: dfe_graduatetalentpoolsearch
+whitehall_slug: department-for-education
+homepage: https://www.gov.uk/guidance/national-minimum-wage-work-experience-and-internships#what-counts-as-an-internship
+tna_timestamp: 20190724054821
+host: https://graduatetalentpoolsearch.direct.gov.uk/gtp/index
```

```suggestion
host: graduatetalentpoolsearch.direct.gov.uk
```

Any page under that domain will be transitioned, so I think it only needs to be set to this.